scripts/parallellize_cargo_check.py

```diff
@@ -34,4 +32,8 @@
     instance_idx * 2 * task_per_core : (instance_idx + 1) * 2 * task_per_core
 ]:
     print("Running ", task)
-    cargo_check = subprocess.check_output(task, shell=True, text=True)
+    print(os.environ)
+    if "libafl_frida" in task:
+        cargo_check = subprocess.check_output(task, shell=True, text=True, env=dict(os.environ, DOCS_RS="1"))
```

Comment:
Might be a good idea to comment why we do this for future reference
(`# DOCS_RS is needed for libafl_frida to build without auto-download feature`)